### PR TITLE
fix(quickstart): jenkinsfile is always overriden for quickstart

### PR DESCRIPTION
**Pull request by corinnekrych**
Fixes https://github.com/fabric8io/fabric8-generator/issues/60

### CODE REVIEW

File: src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java [hunk 1]
```diff
@@ -217,6 +216,18 @@ public String convert(PipelineDTO pipeline) {
         LOG.debug("initializeUI took " + watch.taken());
     }
 
+    private boolean isImportRepositoryFlow(Map<Object, Object> attributeMap) {
+        Object object = attributeMap.get(BoosterDTO.class); // Booster is a step for Quickstart flow only
+        if (object == null) {
```
Comment (bartoszmajsak) [inline src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java]: can be one liner `return attributeMap.containsKey(BoosterDTO.class)`

File: src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java [hunk 2]
```diff
@@ -298,15 +309,19 @@ public Result execute(UIExecutionContext context) throws Exception {
...
+                                // the user in import wizard flow has not opt out for the override
                                 File newFile = new File(basedir, ProjectConfigs.LOCAL_FLOW_FILE_NAME);
                                 Files.writeToFile(newFile, pipelineText.getBytes());
                                 LOG.debug("Written Jenkinsfile to " + newFile);
                             }
                         }
                     }
                 }
-                if(overrideJenkinsFile.getValue() == true) {
+                if((isImportRepositoryFlow(attributeMap) && overrideJenkinsFile.getValue() == true)
```
Comment (bartoszmajsak) [inline src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java:319]: `overrideJenkinsFile.getValue() == true` isn't `true` here redundant?

Comment (corinnekrych) [reply src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java:319]: not redundant, we need to check this boolean as the user can opt to not override the jenkins file in the case of import flow only.
but maybe you meant i could just remove `== true`, I guess yes but i find it clearer that way.

File: src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java (hunk 1 above)
Comment (corinnekrych) [reply src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java]: oki, let me revisit that one.

**Review by jstrachan — APPROVED** [review]
LGTM

**Pull request merged**